### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by suzannepk**
Checking to see if I can edit this way to get things up quickly.

## Checklist:

- [x] I have posted the link for the PR in the hpcwdr slack (#website) to ask for reviewers
- [x] I have previewed changes locally
- [x] I have updated the README.md if necessary

cc @hpcwdr-maintainers

### CODE REVIEW

File: pages/about.md [hunk 1]
```diff
@@ -9,19 +9,19 @@ permalink: /about/
 
 ### Who we are
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+The High Performance Computing (HPC) Workforce Development and Retention (WDR) Action group was established by the  Exascale Computing Project’s Broadening Participation Initiative as part of a sustainable plan to recruit and retain a diverse HPC workforce by fostering a supportive and inclusive culture within the computing sciences at DOE national laboratories. The large muti-lab collaboration of ECP serves as a good starting point for collecting and disseminating best practices for workforce development and retention, and cultural development.
```
Comment (williamfgc) [inline pages/about.md:12]: US Department of Energy (DOE) Exascale Computing Project (ECP)

Comment (williamfgc) [reply pages/about.md:12]: ECP and DOE acronym used before definition

Comment (williamfgc) [reply pages/about.md:12]: multi-lab (typo)

File: pages/about.md [hunk 2]
```diff
@@ -9,19 +9,19 @@ permalink: /about/
...
+
 
 ### Why we formed
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+The Department of Energy’s Advanced Scientific Computing Research Program noted that the DOE National Laboratories face workforce recruitment and retention challenges in the fields within the Computing Sciences that are relevant to their missions.  Future projections indicate an increasing workforce gap and a continued underrepresentation of minorities and females in the workforce unless there is an intervention.
```
Comment (williamfgc) [inline pages/about.md:17]: The DOE's Advanced Scientific Computing Research Program (ASCR) noted that National Laboratories

File: pages/about.md (hunk 1 above)
Comment (williamfgc) [reply pages/about.md:12]: The High Performance Computing (HPC) Workforce Development and Retention (WDR) - HPCWDR -

**Review by williamfgc — CHANGES_REQUESTED** [review]
Sounds great, mostly changes related to defining acronyms on first use.

**Review by williamfgc — APPROVED** [review]

**Comment by suzannepk** [comment]
the Acronym to English fix is a good one.

**Pull request merged**